.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/libs/lvgl"]
+	path = src/libs/lvgl
+	url = https://github.com/joaquimorg/lvgl.git
```

Comment:
What is the reason to use your fork instead of main repo?

Reply:
The main repo don't have the function "lv_disp_set_direction"